### PR TITLE
sunset and adjust the cron time for ROSA with OCP e2e testing

### PR DESCRIPTION
**Pull request by yasun1**
[OCM-9674](https://issues.redhat.com//browse/OCM-9674)

### CODE REVIEW

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-stable.yaml
Comment (radtriste) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-stable.yaml:1]: I see still some jobs with 14 days. Example: `aws-rosa-hcp-stage-critical-f14`

**Review by radtriste — APPROVED** [review]
.lgtm

**Comment by yasun1** [comment]
@liangxia Could you help review the PR? Thanks.

**Pull request merged**